### PR TITLE
Update relationship permissions documentation

### PR DESCRIPTION
**Pull request by aydun**
https://github.com/civicrm/civicrm-user-guide/issues/344

### CODE REVIEW

File: docs/organising-your-data/relationships.md [hunk 1]
```diff
@@ -14,25 +14,38 @@ organisation to the individual.
...
-![image](/img/4.5_Add_Relationship.png)
+![image](/img/5.5_Add_Relationship.png)
 
 1.  Navigate to one of the records that you want to relate.
 2.  Click on **Add Relationship** from the **Actions** button on the
     contact summary, or from the Relationships tab
-3.  Select the Relationship Type. In this case it would be either "Child
-    of" or "Parent of".
+3.  Select the Relationship Type. In this case it would be either "Sibling
+    of".
```
Comment (homotechsual) [inline docs/organising-your-data/relationships.md:23]: If there's one option we should remove the word "either".

**Review by homotechsual — CHANGES_REQUESTED** [review]
Minor text change - otherwise looks good!

File: docs/organising-your-data/relationships.md (hunk 1 above)
Comment (aydun) [reply docs/organising-your-data/relationships.md:23]: removed

**Review by homotechsual — APPROVED** [review]
Superb! This is much clearer!

**Pull request merged**